README.md

```diff
@@ -1,6 +1,44 @@
 # amen
 A toolbox for algorithmic remixing, after Echo Nest Remix.
 
+# Platforms
+Amen is developed on Ubuntu 14.04 and higher.  OS X is probably workable.  Windows users should install Ubuntu.
+
+# Installation
+Amen is pretty simple, but it stands on top of some complex stuff.
+
+First, you'll need our favorite basic resampler, `libsamplerate`  You get get it with `apt-get`, or the package manager of your choice.
+
+Then, you'll need to get our own version of it `pip install git+https://github.com/bmcfee/samplerate.git`
+
+Now, if you're a serious Python nerd, you can just get Amen from pip:  `pip install amen`.
+
+If not, read on for faster, more user-friendly installation process.
+
+First, you should install Anaconda, (https://www.continuum.io/downloads) which will get you all of the dependencies.
+
+Then, install via pip:  `pip install amen`.
+
+# Testing the Installation
+After installation is finished, open up a Python interpreter and run the following (or run it from a file):
+```
+from amen.audio import Audio
+from amen.synthesize import synthesize
+
+audio_file <PATH_TO_YOUR_AUDIO_FILE>
+audio = Audio(audio_file)
+
```

Comment:
Since we're depending on librosa anyway, we could use its bundled example file.

```python
import librosa
audio_file = librosa.util.example_audio_file()
```